build/proj/CoreBuild.proj

```diff
@@ -48,16 +64,18 @@
     We invalidate the cache by changing the value of __BuildPhase property.
   -->
   <Target Name="Restore">
+    <Message Text="Restoring $(SolutionFileName) [$(Configuration)]...%0a" Importance="high" />
     <MSBuild Projects="$(SolutionPath)" Properties="@(SolutionRestoreAndBuildProperty);__BuildPhase=SolutionRestore" Targets="Restore" BuildInParallel="true" />
   </Target>
 
   <Target Name="Build">
+    <Message Text="Building $(SolutionFileName) [$(Configuration)]...%0a" Importance="high" />
     <ItemGroup>
       <SolutionBuildTarget Include="Rebuild" Condition="'$(Rebuild)' == 'true'" />
       <SolutionBuildTarget Include="Build" Condition="'$(Build)' == 'true' and '$(Rebuild)' != 'true'" />
       <SolutionBuildTarget Include="Test" Condition="'$(Test)' == 'true'" />
       <SolutionBuildTarget Include="IntegrationTest" Condition="'$(IntegrationTest)' == 'true'" />
-      <SolutionBuildTarget Include="Pack" Condition="'$(BuildOrRebuild)' == 'true'" />
+      <SolutionBuildTarget Include="Pack" Condition="'$(BuildOrRebuild)' == 'true' AND '$(QuickSrcBuild)' != 'true'" />
```

Comment:
Strangely enough, we don't use the `Pack` property to actually indicate if the Pack target is used. The quick Src build needs to not Pack.